src/Compilers/Test/Resources/Core/Microsoft.CodeAnalysis.Compiler.Test.Resources.csproj

```diff
@@ -85,11 +86,16 @@
     <Content Include="NetFX\aacorlib\Key.snk" />
     <EmbeddedResource Include="NetFX\Minimal\minasync.dll" />
     <EmbeddedResource Include="NetFX\Minimal\mincorlib.dll" />
+    <EmbeddedResource Include="NetFX\PortableProfile7\mscorlib.dll" />
+    <EmbeddedResource Include="NetFX\PortableProfile7\System.Runtime.dll" />
```

Comment:
Attempted to add a portable7 variant to Basic.Reference.Assemblies but these aren't available through NuPkg anywhere. Decided to just use them directly as resources.